pages/index.js

```diff
@@ -89,18 +86,25 @@ export default function Home({ isConnected }) {
                 id={habit.id} 
                 isEditable={true} 
                 color={cardColors[index % cardColors.length]} 
-                habit={habit} setHabits={setHabits} 
+                habit={habit} 
+                setHabits={setHabits}
+                completedHabits={completedHabits} 
```

Comment:
Why does each single habit component need to know the total completed habits here?

Reply:
Ah i see, it looks like you're using it in the child to update the count of completed habits. You could potentially update the parent compnent here to

` setCompletedHabits={() => setCompletedHabits(completedHabits +1) }`

which would let you delete the logic you're doing in the individual Habit component. Ideally the Habit component is "dumb" and doesn't need to care about the outside world / maintaining state of the entire app, if that makes sense.